### PR TITLE
Feedback invalid index via default parseIndex() exception instead of invalide format error

### LINKED ISSUE #139: [Alek] `screen job [non natural number]` error message is not specific
- when 0 is used as index, the feedback does not make it clear that it is the index part that is the problem
- Same goes for non- numbers `asdf` and negative numbers `-1`

image

### PR DESCRIPTION
**Pull request by KengHian**
- closes #139 
image

Do you think invalid index such as "-1" or "one" should throw invalid format error or feedback "index is not non-zero integer"?

### CODE REVIEW

**Review by wassilililily — APPROVED** [review]
LGTM, I think changing to this new error message is okay but might not be a high priority fix

**Comment by KengHian** [comment]
> LGTM, I think changing to this new error message is okay but might not be a high priority fix

Sure, just add an issue regarding this and we remember to do this at a later time.

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java [hunk 1]
```diff
@@ -34,13 +33,8 @@ public DeleteCommand parse(String args) throws ParseException {
...
         String indexString = splitArgs[1];
 
-        Index index;
-        try {
-            index = ParserUtil.parseIndex(indexString);
-        } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE), pe);
-        }
+        Index index = ParserUtil.parseIndex(indexString);;
```
Comment (Green-Tea-123) [inline src/main/java/seedu/address/logic/parser/DeleteCommandParser.java:36]: 2 semicolons?

**Review by Green-Tea-123 — APPROVED** [review]
Overall LGTM.

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java (hunk 1 above)
Comment (KengHian) [reply src/main/java/seedu/address/logic/parser/DeleteCommandParser.java]: Opps, fixed!

**Pull request merged**